conftest.py

```diff
@@ -267,6 +270,9 @@ def message_forwarder(messaging_ports):
         args.append(str(pub))
 
     proc = subprocess.Popen(args, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+    # It takes a while for the forwarder to start, so allow for that.
+    # TODO(jamessynge): Come up with a way to speed up these fixtures.
+    time.sleep(3)
```

Comment:
Were we missing messages in some of the tests?

Reply:
Only for tests that I'd written, and was trying to add in another PR that I'll send soon for coverage of the arduino i/o code. I'm going to try to improve the msg forwarding support in the tests in general a bit later.